marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/ingestion/bulkIngester.sjs

```diff
@@ -29,6 +29,29 @@ const consts = require('/data-hub/5/impl/consts.sjs');
 const state = fn.head(xdmp.fromJSON(endpointState));
 
 const work = fn.head(xdmp.fromJSON(workUnit));
+let collections=[];
+if(work.collections !=null){
+  collections=work.collections.split(',');
+}
+
+const permissionsArray = []
+
+if(!work.permissions){
+  work.permissions='data-hub-common,read,data-hub-common,update'
+}
+
+const permissions = work.permissions.split(',');
+
+if((permissions.length%2)!= 0){
+  xdmp.log("Error: Unable to parse Permissions")
```

Comment:
No need for xdmp.log here, the stacktrace from the fn.error will be visible in the logs